spec/values/calculation/round/strategy/README.md

```diff
@@ -0,0 +1,5 @@
+### Three-argument round strategy testing
+
+Tests the `round` function with the various combinations of rounding modes to cover all possibilities and edge cases to ensure robustness. Some tests are based on [web-platform-tests] for the CSS round function.
+
+[web-platform-tests]=https://github.com/web-platform-tests/wpt/blob/master/css/css-values/round-function.html
```

Comment:
Wrong syntax for markdown link references. There's syntax documentation [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).